Formula/govendor.rb

```diff
@@ -17,7 +17,7 @@ class Govendor < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "24497503629e520a1fe718029ab520b05c012677df065c9fd104afe4d898b8b8"
   end
 
-  deprecate! date: "2020-03-02", because: :repo_archived
+  disable! date: "2022-07-28", because: :repo_archived
```

Comment:
`dockviz` still uses this to build. Last time we checked, the project was supposedly still maintained https://github.com/justone/dockviz/issues/49 and was going to migrate to go modules (comment from ~1 year ago)